docs/uart.rst

```diff
@@ -6,6 +6,8 @@ UART
 The ``uart`` module lets you talk to a device connected to your board using
 a serial interface.
 
+.. note::
+    The internal uart buffer is hard coded to 64 bytes.
```

Comment:
Could we change this to `The internal UART RX buffer is 64 bytes, so make sure data is read before the buffer is full or some of the data might be lost`.

It clarifies a bit what having a fix size buffer means in this case.